maestro-orchestra-models/src/main/java/maestro/orchestra/Commands.kt

```diff
@@ -801,6 +801,42 @@ data class RepeatCommand(
 
 }
 
+data class RetryCommand(
+    val maxAttempts: String? = null,
```

Comment:
Instead of calling this maxAttempts we'll probaby call this `maxRetries`. Feedback from DuckDuckGo.